stories/input.stories.js

```diff
@@ -587,6 +658,35 @@ storiesOf('Low level blocks/Inputs', module)
     );
   });
 
+const H1 = styled.h1`
+  font-size: 3em;
+  margin-top: 1em;
+`;
+
+const H2 = styled.h2`
+  margin-top: 0.5em;
+  font-size: 2em;
+`;
+
+const P = styled.p`
+  font-size: 1.5em;
+  margin: 0.5em 0;
```

Comment:
why the sudden change to em?

Reply:
ow my bad, I normally use EM outside wegroup, wasn't thinking for a sec